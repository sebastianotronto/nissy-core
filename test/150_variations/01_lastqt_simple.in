lastqt
U2 D L' F
