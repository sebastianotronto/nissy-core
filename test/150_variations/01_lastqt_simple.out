2
U2 D L' F
U2 D L' F'
